testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/micrometer/BasicMicrometerTestCase.java

```diff
@@ -0,0 +1,50 @@
+package org.wildfly.test.integration.observability.micrometer;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import jakarta.inject.Inject;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.testcontainers.api.DockerRequired;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.test.shared.CdiUtils;
+import org.jboss.as.test.shared.util.AssumeTestGroupUtil;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.AssumptionViolatedException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.test.integration.observability.opentelemetry.application.JaxRsActivator;
+
+
+@RunWith(Arquillian.class)
+@ServerSetup(MicrometerSetupTask.class)
+@DockerRequired(AssumptionViolatedException.class)
+public class BasicMicrometerTestCase {
+    @Inject
+    private MeterRegistry meterRegistry;
+
+    // The @ServerSetup(MicrometerSetupTask.class) requires Docker to be available.
+    // Otherwise the org.wildfly.extension.micrometer.registry.NoOpRegistry is installed which will result in 0 counters,
+    // and cause the test fail seemingly intermittently on machines with broken Docker setup.
+    @BeforeClass
+    public static void checkForDocker() {
+        AssumeTestGroupUtil.assumeDockerAvailable();
+    }
```

Comment:
Given the `@DockerRequired(AssumptionViolatedException.class)`, this should not be needed.